pkg/metricprovider/custom_metric_provider.go

```diff
@@ -70,89 +70,95 @@ func (p *CustomMetricProvider) GetMetricByName(ctx context.Context, name types.N
 func (p *CustomMetricProvider) GetMetricBySelector(ctx context.Context, namespace string, selector labels.Selector, info provider.CustomMetricInfo, metricSelector labels.Selector) (*custom_metrics.MetricValueList, error) {
 	klog.Info(fmt.Sprintf("Get metric by selector for custom metric, Info %v namespace %s selector %s metricSelector %s", info, namespace, selector.String(), metricSelector.String()))
 
-	if !IsLocalCustomMetric(info, p.client) {
```

Comment:
don't remove LocalCustomMetric part

Reply:
In this scenario, the metric name of craned is already set to constant， for example: known.MetricNamePrediction="crane_autoscaling_prediction"
If it matches, it's a local metric，